src/tastyscrapy/spiders.py

```diff
@@ -48,18 +49,46 @@ def on_login_response(self, response):
         if xjpath.path_lookup(payload, 'status') != ('success', True):
             raise LoginFailedException("Unable to log in using provided credentials.")
 
-        # emit an item which shows that we logged in successfully for audit tracking
-        yield LoginSuccess(created=datetime.utcnow().isoformat())
-
+        # lookup redirect url
         redirect, is_success = xjpath.path_lookup(payload, 'redirect')
 
+        # extract session key for cookie
+        session_key, session_value = xjpath.XJPath(payload)['session'].split('=')
```

Comment:
A little sneaky thing that Delicious does is it returns its session cookie in the JSON response in the `session` key and is formatted like `delavid=SESSIONID`, so we split it on the equals sign.